spec/support/spree/order_helper.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Spree
+  module OrderHelpers
+    # This method is used to advance an order through the state machine until it is completed.
+    def advance_order(order)
+      while !order.completed?
+        break unless order.next!
+      end
+    end
+  end
+end
```

Comment:
Did you see the `OrderWorkflow` service? I think it would make sense to re-use that code instead of creating our own helper which does only part of what's happening in a real checkout.

Reply:
Good pickup, it looks like `OrderWorkflow::complete!` can be used directly in place of this.

Reply:
done